wremnants/datasets/datasetDict2018_v9.py

```diff
@@ -0,0 +1,183 @@
+        'group': "Zmumu",
+        "das_name" : "private"
+    },    
+    'DYJetsToMuMuMass10to50PostVFP' : {
+        'filepaths' : ["{BASE_PATH}/../y2018/BKGV9/DYJetsToMuMu_M-10to50_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/NANOAODSIM/106X_upgrade2018_realistic_v16_L1v1-v1/*/*.root",
+                       "{BASE_PATH}/../y2018/BKGV9/DYJetsToMuMu_M-10to50_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/NANOAODSIM/106X_upgrade2018_realistic_v16_L1v1_ext1-v1/*/*.root"],
+        'xsec' : xsec_ZmmMass10to50PostVFP,
+        'group': "DYlowMass",
+        "das_name" : "/DYJetsToMuMu_M-10to50_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/RunIISummer20UL18NanoAODv9-106X_upgrade2018_realistic_v16_L1v1*v1/NANOAODSIM"
+    },
+    #'ZtautauPostVFP' : { #this sample needs to be produced
+    #    'filepaths' : ["{BASE_PATH}/../y2018/DYJetsToTauTau_M-50_AtLeastOneEorMuDecay_massWgtFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/NANOAODSIM/106X_upgrade2018_realistic_v16_L1v1-v2/*/*.root"],
+    # At least one tau->e or mu decay, so everything that's not all other decays
+    #   'xsec' : xsec_ZmmPostVFP*Z_TAU_TO_LEP_RATIO,
+    #'group': "Ztautau",
+    #  'group': "DYlowMass",
+    #  "das_name" : "/DYJetsToTauTau_M-50_AtLeastOneEorMuDecay_massWgtFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/RunIISummer20UL18NanoAODv9-106X_upgrade2018_realistic_v16_L1v1-v2/NANOAODSIM"
+    #},
+
+    'WplusmunuPostVFP' : { 
+        'filepaths' : ["{BASE_PATH}/../y2018/WplusJetsToMuNu_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/NanoV9MC2018_2018test/230530_220442/*/*.root"],
+        'xsec' : xsec_WpmunuPostVFP,
+        'group': "Wmunu",
+        "das_name" : "private"
+    },
+    'WminusmunuPostVFP' : { 
+        'filepaths' : ["{BASE_PATH}/../y2018/WminusJetsToMuNu_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/NanoV9MC2018_2018test/230530_220636/*/*.root"],
+        'xsec' : xsec_WmmunuPostVFP,
+        'group': "Wmunu",
+        "das_name" : "private"
+    },
+    'WplustaunuPostVFP' : { 
+                         'filepaths' : 
+                         ["{BASE_PATH}/../y2018/WplusJetsToTauNu_TauToMu_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/*.root",],
+                         'xsec' : BR_TAUToMU*xsec_WpmunuPostVFP,
+                         'group': "Wtaunu",
+                         "das_name" : "private"
+    },    
+    'WminustaunuPostVFP' : { 
+                         'filepaths' : 
+                         ["{BASE_PATH}/../y2018/WminusJetsToTauNu_TauToMu_H2ErratumFix_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/*.root"],
+                         'xsec' : BR_TAUToMU*xsec_WmmunuPostVFP,
+                         'group': "Wtaunu",
+                         "das_name" : "private"
+    },
+    'TTLeptonicPostVFP' : { 
+        'filepaths' : ["{BASE_PATH}/../y2018/BKGV9/TTTo2L2Nu_TuneCP5_13TeV-powheg-pythia8/NANOAODSIM/106X_upgrade2018_realistic_v16_L1v1-v1/*/*.root"],
+        'xsec' : 88.29,
```

Comment:
Also these cross sections values could be defined somewhere central, e.g. utilities.common